docs/source/user/setup.rst

```diff
@@ -181,6 +181,10 @@ You can review the discovery metadata just cached through the new item at  ``/oa
 
 Repeat this step for any other discovery metadata you wish to publish, such as the ``temp`` dataset.
 
+.. note::
+
+   You must ensure that your discovery metadata is published once the WMO Global Broker is connected and subscribed for your wis2box broker.  If you have published discovery metadata before this stage, you must re-publish using the command above.
```

Comment:
Add prefix "To ensure your discovery metadata is shared with the Global Discovery Catalogue, you must ensure that ... "

Reply:
Fixed.